keyboards/bobpad/keymaps/via/keymap.c

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2021 Ananya Kirti
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include QMK_KEYBOARD_H
+
+
+bool is_alt_tab_active = false;
+uint16_t alt_tab_timer = 0;
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_MEDIA_STOP,  KC_MEDIA_PLAY_PAUSE, KC__MUTE,
+        KC_MEDIA_REWIND,  KC_MEDIA_FAST_FORWARD,  KC_PGDN
+    ),
+    [1] = LAYOUT(
+        _______,  _______,  _______,
+        _______,  _______,  _______
+    ),
+    [2] = LAYOUT(
+        _______,  _______, _______,
+        _______,  _______, _______
+    ),
+    [3] = LAYOUT(
+        _______,  _______, _______,
+        _______,  _______, _______
+    ),
+    [4] = LAYOUT(
```

Comment:
This layer will not be visible in VIA. You need to either remove it or add `#define DYNAMIC_KEYMAP_LAYER_COUNT 5` in config.h.

Reply:
removed the layout, thanks!